<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectManager</class>
 <widget class="QMainWindow" name="projectManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Paramters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,0,0,0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="projName_le"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Customer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cutomerName_le"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Deadline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deadline_de">
           <property name="displayFormat">
            <string>d/M/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Project manager</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="projManagerName_le">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Project Manager email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="projManagerEmail_le"/>
       </item>
       <item>
        <widget class="QLabel" name="info_lb">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>ТЕКСТ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="settings_btn">
         <property name="text">
          <string>settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cleanup_btn">
         <property name="text">
          <string>cleanup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="arch_btn">
         <property name="text">
          <string>archive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="create_btn">
         <property name="text">
          <string>create</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="proj_list"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
